tutorials/07a_Advanced_ChangeStartingScripts.md

```diff
@@ -0,0 +1,66 @@
+# The REMIND starting scripts
+
+David Klein (<dklein@pik-potsdam.de>), Lavinia Baumstark (<baumstark@pik-potsdam.de>), Mika Pflüger (<mika.pflueger@pik-potsdam.de>)
+
+## Introduction
+
+Before GAMS can start solving the model, several steps are necessary. The input data needs to be acquired and put into the right places, the model needs to be compiled in the chosen configuration, and the model needs to be started. These preparatory tasks are done in R in the scripts `start.R`, `start_coupled.R`, `start_bundle_coupled.R` in the main folder using functions from the `scripts/start` folder.
+
+## The starting procedure
+
+### Overview
+
+```
+start.R -> submit(cfg) -------------------------> prepare_and_run()
+
+           - create output folder                 - fetch input data
+           - copy config, prepare_and_run.R       - prepare NDCs
+             to output folder                     - create full.gms
+           - send slurm job to cluster            - run GAMS
+                                                  - reporting
+
+|-----------------------------------------------|--------------------|
+                  login node                           slurm job
+```
+
+### Detail
+
+```
+Rscript start.R
+  -> choose_slurmConfig()               [scripts/start/choose_slurmConfig.R]
+     configure_cfg(cfg, scenario, ...)  [start.R]
+     save cfg to runtitlte.RData (in REMIND's mainfolder)
+     submit(cfg)                        [scripts/start/submit.R]
+       - create output folder
+       - copy scripts/start/prepare_and_run.R into results folder
+       - save cfg to config.RData into results folder
```

Comment:
it's actually `config.Rdata` with a small `d`, as opposed to `runtitle.RData` which actually uses a capital `D`. Why ever.

Reply:
Thanks, your attention to detail is appreciated!